packages/caver-kct/src/kip17Token.sol

```diff
@@ -0,0 +1,2980 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @dev Minting with URI extension of the KIP7 standard as defined in the KIP.
+ * See https://kips.klaytn.com/KIPs/kip-17#minting-with-uri-extension
+ */
+interface IKIP17MetadataMintable {
+    /**
+     * @dev Creates `tokenId` token with a link `tokenURI`and assigns them to `account`
+     * increasing the total supply.
+     *
+     * Requirements:
+     *
+     * - caller must have the {KIP7Mintable-MINTER_ROLE}
```

Comment:
hmm? it should also be updated KIP7 with KIP-17 in the original repo?

Reply:
this code is from IKIP17MetadataMinatable.sol from original repo. 
it looks like the original repo needs to be updated, too.

Reply:
yeah, first of all I think we can update it in this file and the original repo separately.